core/amber/src/main/python/core/runnables/network_sender.py

```diff
@@ -59,15 +59,17 @@ def _send_data(self, to: ActorVirtualIdentity, data_payload: DataPayload) -> Non
                 schema=data_payload.schema,
             )
             data_header = PythonDataHeader(tag=to, is_end=False)
-            self._proxy_client.send_data(bytes(data_header), table)
+            returned_credits = self._proxy_client.send_data(bytes(data_header), table)
 
         elif isinstance(data_payload, EndOfUpstream):
             data_header = PythonDataHeader(tag=to, is_end=True)
-            self._proxy_client.send_data(bytes(data_header), None)
+            returned_credits = self._proxy_client.send_data(bytes(data_header), None)
 
         else:
             raise TypeError(f"Unexpected payload {data_payload}")
 
+        # print('Python sent data message, received back: ' + repr(returned_credits))
```

Comment:
before I read the codes, can you please clean all debugging related code (i.e., prints) and unnecessary comments? Just leave necessary changes.

Reply:
sounds good, jjust updated